defender/utils.py

```diff
@@ -17,63 +18,37 @@
 
 def is_valid_ip(ip_address):
     """ Check Validity of an IP address """
-    valid = True
-    try:
-        socket.inet_aton(ip_address.strip())
-    except (socket.error, AttributeError):
-        valid = False
-    return valid
+    if not ip_address:
+        return False
+    ip_address = ip_address.strip()
+    ipv4_re = r'^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$'
+    ipv6_re = r'\[[0-9a-f:\.]+\]'
+    ipv4_regex = re.compile(ipv4_re)
+    ipv6_regex = re.compile(ipv6_re)
```

Comment:
What do you think about just using django.core.validators.validate_ipv46_address directly?

Then you could replace the code with:
```python
try:
    validate_ipv46_address(ip_address)
    return True
except ValidationError:
    return False
```